workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ResourceRequirements/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import FormInput from '../../../components/Input/formInput';
+import i18n from '../../../i18n';
+import {useAppContext} from '../../../manage-app-state/AppManageState';
+import {TYPES} from '../../../manage-app-state/actionTypes';
+
+import './index.scss';
+
+export type ResourceRequirementsForm = {
+	numberOfCPUs: number;
```

Comment:
I think you forgot to change

```suggestion
	cpus: number;
```

Reply:
hmmm, I do not need this anymore. I'm gonna remove it.